docs/user-guides/json-pattern-matching-authorization.md

```diff
@@ -124,19 +124,19 @@ spec:
       endpoint: http://keycloak.keycloak.svc.cluster.local:8080/auth/realms/kuadrant
   authorization:
   - name: email-verified-only
+    conditions:
```

Comment:
maybe not a good idea but will throw it out there: What if this were named: ```applyWhen``` do you think it would me more intuative? Perhaps not just a thought

Reply:
If you say it's an improvement and not subject to the possible misunderstanding I mention in my 2nd point in https://github.com/Kuadrant/authorino/pull/206#issuecomment-1011031494, sounds good to me too

To sum up a few options we've got so far:
- `conditions`
- `when`
- `applyWhen`
- `if`

I personally prefer single-word names whenever possible, but only when they are clear enough.

Let's start a quick poll?

Reply:
Yes as an improvement. Single word is fine also. ```When``` gets my vote

Reply:
Two votes for `when`. Sounds like a crushing majority to me 🙂 I'll make the changes.

Thanks!